ci/build.ps1

```diff
@@ -115,24 +115,25 @@ cmake --build . --config $cmakeBuildType -- $cmakeGeneratorArgs ; exitIfFailed
 
 # Functional tests
 # The $LastExitCode from MSBuild can't be trusted
+$env:TEST_FILE = "test/functional/core/job_spec.lua"
```

Comment:
You can also specify `TEST_FILTER` (to not having to comment other tests).
Also `BUSTED_ARGS=--repeat=100` might be good.